torch/optim/optimizer.py

```diff
@@ -260,6 +260,7 @@ def __init__(self, params: ParamsT, defaults: Dict[str, Any]) -> None:
                                "is deprecated. In the future, this will raise an error. "
                                "Please wrap your Tensor in an iterable instead."),
                               FutureWarning)
+                params = [params]
```

Comment:
Without this line, 272 will torch.unbind, which is not what we want

Reply:
I'm p sure we have an override here https://github.com/pytorch/pytorch/blob/main/torch/_tensor.py#L1004